titus-ext/jooq/src/main/java/com/netflix/titus/ext/jooq/JooqJobActivityPublisherStore.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package com.netflix.titus.ext.jooq.jobactivity;
```

Comment:
Are we meaning to remove the scoped package?

Reply:
nope didnt mean to, moving it back. good catch